### ISSUE TITLE
Cannot open in Delphi 2007

### ISSUE THREAD

**Issue by Kortenbach** [pjde/delphi-mqtt#5]
When opening either the .dpk or the dproj file in D2007 I get:

unable to load project
...MQTTComponents.dproj
Only one top level element is allowed in an XML 
Document.

Line: 2
<Delphi.personality><Source><Source Nam

Googled a bit:
- I should delete the dproj and use the dpr instead. However: the dpr is not included in the archive.

**Comment by pjde** [comment]
The MQTTComponents are a Delphi package and have a dpk file not dpr.

The last version of this was generated on XE so it could be some 
backwards compatible issue.


You may be able to create your own empty package and import the pascal 
files in.


Regards


Paul

------ Original Message ------

**Comment by Kortenbach** [comment]
Hello pjde,

Thank you very much for the quick reply!
Here is what I tried:
I renamed the dproj file and re-opened the dpk. This time no error!

I do however have a new problem now (that I did not investigate very thoroughly yet):
_[DCC Error] MQTTComponents.dpk(30): E2202 Required package 'OverbyteIcsDXeRun' not found_
I did install ICS Design-time package... (do I need to install 'OverbyteIcsDXeRun' separately?)
Will keep you posted...
